packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -213,6 +218,19 @@ const buildAPIProject = async (
   return builtProject;
 };
 
+const getRDSLayerMapping = async (): Promise<RDSLayerMapping> => {
+  try {
+    const response = await fetch(LAYER_MAPPING_URL);
```

Comment:
Should we be explicitly importing node-fetch instead of using native `fetch`?

Reply:
Since we are using node18 now, I don't see a problem with using native fetch.